src/insert.rs

```diff
@@ -78,19 +78,16 @@ impl SupabaseClient {
     ///
     ///
     /// # Returns
-    /// This method returns a `Result<String, String>`. On success, it returns `Ok(String)` with the new row's ID,
+    /// This method returns a `Result<(), String>`. On success, it returns `Ok(())`,
     /// and on failure, it returns `Err(String)` with an error message.
-    pub async fn insert(&self, table_name: &str, mut body: Value) -> Result<String, String> {
+    pub async fn insert(&self, table_name: &str, body: Value) -> Result<(), String> {
         let endpoint: String = format!("{}/rest/v1/{}", self.url, table_name);
 
         #[cfg(feature = "nightly")]
         use crate::nightly::print_nightly_warning;
         #[cfg(feature = "nightly")]
         print_nightly_warning();
 
-        let new_id: i64 = generate_random_id();
```

Comment:
I see that `generate_random_id` is not used anywhere else.
Should I remove it ?